### PR TITLE
Call to SpacyMatcher into Lambda_function.py

### PR DESCRIPTION
**Pull request by ANA-POTJE**
Changes in the lambda_function.py, including in the JSON object the "spacyMatcher.py" output data - new type 'topics':
       'results' : [
         { 'type' : 'credibility' ...... },
         { 'type' : 'polarity' ..... },
         { 'type' : 'objectivity' .....},
         { 'type' : 'biasscore' .....}
         { 'type' : 'topics' ,'outcome': {'Today': 'DATE', 
                                                        'Donald Trump': 'PERSON',
                                                        'USA': 'GPE',
                                                        'Toyota': 'ORG'}}

### CODE REVIEW

File: src/lambda_function.py [hunk 1]
```diff
@@ -110,6 +115,25 @@ def lambda_handler(event, context):
...
+    if sentanalysisresult['text'] != '-1':
+        try:
+            dict_tot = {}
+            tags = ['PERSON','GPE','ORG','PERCENT','LANGUAGE','DATE','TIME','LOC','NORP','EVENT','WORK_OF_ART',
+                    'MONEY','QUANTITY','ORDINAL','CARDINAL']
+            for i in tags:
+                list_objs = sm.spacyMatcher(sentanalysisresult['text'], i) ### {'Toyota', 'BBC', 'BMW', 'Chanel'}
+                for obj in list_objs:
+                    dict_tot[obj] = i                    
+            object['results'].append({'type': 'topics', 'outcome': dict_tot})
```
Comment (jsjohnstone) [inline src/lambda_function.py:129]: Can I suggest we move this to the 'article' object, and bundle each topic/type as a dict?

```
article_topics = []
for i in tags:
                list_objs = sm.spacyMatcher(sentanalysisresult['text'], i) ### {'Toyota', 'BBC', 'BMW', 'Chanel'}
                for obj in list_objs:
                    article_topics.append({'type' : i, 'topic' : obj})

object['article']['topics'] = article_topics
```

...which should return an object like:

```
{
  "url": ...
  "results": ...
  "article": {
    "header":...
    "topics":[
      {"type":"PERSON", "topic":"Boris Johnson"},
      {"type":"GPE","topic":"United Kingdom"}
    ]
  }
}
```

Comment (jsjohnstone) [reply src/lambda_function.py:129]: I think this'll be easier to process on the frontend, as we can do a .map() on the topics list, and then pass the topic object (`{"type":"PERSON", "topic":"Boris Johnson"}`) to the chip to display.

**Review by jsjohnstone — CHANGES_REQUESTED** [review]

File: src/lambda_function.py (hunk 1 above)
Comment (ANA-POTJE) [reply src/lambda_function.py:129]: That is not a problem ...! 
Can I please ask you to first fix the error: "E       OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory."
I guess we need to install the following module in AWS:
=> python -m spacy download en_core_web_sm

Comment (jsjohnstone) [reply src/lambda_function.py:129]: I've just made a commit (e86d604) which add's the install command to the build_script.sh... just need to see if it works! :)

**Comment by ANA-POTJE** [comment]
Closing this PR as the new PR #84 contains the changes required.

**Pull request closed without merging**